LabExT/View/AddonSettingsDialog.py

```diff
@@ -181,6 +195,15 @@ def _get_loaded_lvcards_and_paths(self):
                 ret_list.append(('    ->', '', vi))
         return ret_list
 
+    def _get_loaded_stages_and_paths(self) -> list:
```

Comment:
Now the Stages path from the core (.../LabExT/Movement/Stages) is listed in the addons list as default. Can we get rid of this, like it is done for the instruments and measurements?

Reply:
I removed it. Now it looks like this:
![Screenshot 2023-02-27 at 12 27 09](https://user-images.githubusercontent.com/14354617/221552280-3dff100f-460f-46a6-ad2b-495bc0994505.png)